archlinuxcn-mirrorlist

```diff
@@ -1,106 +1,112 @@
 ##
-## Arch Linux CN community repository mirrorlist
-## Generated on 2023-08-25
+## Arch Linux CN community Repository mirrorlist
+## Generated on 2023-12-09
 ##
 
-## Our main server (Amsterdam, the Netherlands) (ipv4, ipv6, http, https)
+## Our main server (Amsterdam, North Holland, Netherlands) (http, https, ipv4, ipv6)
 # Server = https://repo.archlinuxcn.org/$arch
 
-## CERNET (中国) (ipv4, ipv6, http, https)
+## 校园网联合镜像站 (中国) (ipv4, ipv6)
+## Redirect to suitable educational mirror based on location
```

Comment:
Since this mirror site does not have a real path, all the supported protocols work as expected.